src/blocks/content-helper/content-helper.scss

```diff
@@ -13,12 +13,19 @@
 
 .parsely-top-posts {
 	list-style-type: none;
-	margin: 0;
+	margin: 22px 0 0;
```

Comment:
Do you have a specific reason for choosing `px` instead of a unit like `rem`s?

Reply:
No specific reason, let me know if there is a perferred choice in the codebase.

Reply:
Usage of `rem` is preferred whenever this possible.